payplug/resources.py

```diff
@@ -62,13 +63,15 @@ def factory(data):
             exceptions.UnkownAPIResource when it's impossible to reconstruct the APIResource from its data.
         """
         if 'object' not in data:
-            raise exceptions.UnknownAPIResource('Missing `object` key in resource.')
+            raise exceptions.UnknownAPIResource(
+                'Missing `object` key in resource.')
```

Comment:
We usually allow 120 characters columns in our codebase, could you revert these aesthetic changes?

Reply:
I reverted all code formatting changes